cmd/bucket-notification-utils.go

```diff
@@ -259,15 +259,14 @@ func validateNotificationConfig(nConfig notificationConfig) APIErrorCode {
 // - kafka
 // - webhook
 func unmarshalSqsARN(queueARN string) (mSqs arnSQS) {
-	mSqs = arnSQS{}
 	strs := strings.SplitN(queueARN, ":", -1)
 	if len(strs) != 6 {
-		return mSqs
+		return
```

Comment:
Am investigating this in a little wider perspective (impact on actual code that is generated), and in the light of that it may make sense to change how we use named return variables and/or return statements from functions.

Stay tuned for more info.